deploy/openshift/configmap.yaml

```diff
@@ -16,6 +16,9 @@ data:
         url: ${JAEGER_URL}
       grafana:
         url: ${GRAFANA_URL}
+    auth:
```

Comment:
as per my earlier comment, I think this needs to be under a "openshift_config" main group. Otherwise this generically named "auth" group looks like it could be valid for Kubernetes environments too, which AFAIU it is not.

Reply:
Same as the other comment.